content/articles/building-a-payroll-system-with-nextjs/index.md

```diff
@@ -0,0 +1,262 @@
+In this tutorial, we learn basics of Next.js and how we can use them effectively to build a payroll system. 
```

Comment:
Can you add an intro about NextJS? Where we use it? How it is better than React? What is it?